### PR TITLE
Adding the Fargate Lab Walkthrough

### PR DESCRIPTION
**Pull request by CarmenAPuccio**
Why:
This is the userguide for the Fargate Lab Walkthrough. 

Signed-off-by: Carmen A. Puccio <carmenpuccio@yahoo.com>

### CODE REVIEW

File: modules/container-orchestration/readme.adoc [hunk 1]
```diff
@@ -1,6 +1,354 @@
...
+[source,shell]
+----
+docker push 123456789012.dkr.ecr.us-west-2.amazonaws.com/petstore_postgres:latest
+----
+
+You should see the Docker image being pushed:
+
+[source,shell]
+----
+Admin:~/environment $ docker push 123456789012.dkr.ecr.us-west-2.amazonaws.com/petstore_postgres:latest
```
Comment (mandusm) [inline modules/container-orchestration/readme.adoc]: did you mean to copy the PS1?

File: modules/container-orchestration/readme.adoc [hunk 2]
```diff
@@ -1,6 +1,354 @@
...
+[source,shell]
+----
+docker push 123456789012.dkr.ecr.us-west-2.amazonaws.com/petstore_frontend:latest
+----
+
+You should see the Docker image being pushed:
+
+[source,shell]
+----
+Admin:~/environment $ docker push 123456789012.dkr.ecr.us-west-2.amazonaws.com/petstore_frontend:latest
```
Comment (mandusm) [inline modules/container-orchestration/readme.adoc]: Same here? Do you want to show the PS1?

File: modules/container-orchestration/readme.adoc [hunk 3]
```diff
@@ -1,6 +1,354 @@
...
+22. The service will now start your task. Click *View Service* and you will have to wait for your task to transition to *RUNNING*. Feel free to inspect the logs for your task while you wait.
+
+23. Once the task is running, view the *Details* of the petstore *Service*. Under *Load Balancing* click the *ecs-petsto-petstore* Target Group link to view 
+the Tasks being registerd into the Target Group under *Targets*.
+
+24. Once your Registerd Target is registered as *Healthy* you can view the service through the Application Load Balancer.
+
+25. Click on the *Load Balancing* link on the left hand navigation pane of the EC2 window to view your Application Load Balancer.
+
+26. For the *petstore-lb* copy the *DNS name* and navigate to http://<InsertYourCopiedDNSName>/applcationPetstore to see the petstore application running.
```
Comment (mandusm) [inline modules/container-orchestration/readme.adoc]: Isn't there an AWS CLI command with a one-liner bash command we can give to generate the URL?

File: modules/container-orchestration/readme.adoc [hunk 4]
```diff
@@ -1,6 +1,354 @@
...
+containerize-application_petstore   latest              22f901adffb3        32 minutes ago      626MB
+<none>                              <none>              9b961113e5bf        33 minutes ago      564MB
+postgres                            9.6                 506063568b80        2 days ago          237MB
+maven                               3.5-jdk-7           84a20ec9fab9        3 weeks ago         483MB
+lambci/lambda                       nodejs4.3           6c30c5c1b1e0        6 weeks ago         969MB
+lambci/lambda                       python2.7           377732dd7a1f        6 weeks ago         974MB
+lambci/lambda                       python3.6           acf16b1d5297        6 weeks ago         1.1GB
+lambci/lambda                       nodejs6.10          da301bf4fe34        6 weeks ago         1.02GB
+jboss/wildfly                       11.0.0.Final        6926d48f2e5b        4 months ago        617MB
+----
```
[the remark this answers is not in the document]
Comment (CarmenAPuccio) [reply modules/container-orchestration/readme.adoc]: @christopherhein they must be the defaults from Cloud9 environment.

**Pull request merged**